iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/voter/CountryCodeStrictMatchVoter.java

```diff
@@ -20,6 +20,9 @@
     
     //------------------------ LOGIC --------------------------
     
+    /**
+     * Returns true if countryCode of affiliation and organization are non empty and equal.
```

Comment:
I think it would be better to use "link" more often:
... if {@link Affiliation#getCountryCode} and {@link AffMatchOrganization#getCountryCode} are non-empty and equal.